tasks/src/main/java/io/optional/Task1.java

```diff
@@ -0,0 +1,28 @@
+package io.optional;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Random;
+
+// 1.Создать и заполнить файл случайными целыми числами. Отсортировать содержимое файла по возрастанию.
+public class Task1 {
+  public static void main(String[] args) throws IOException {
+    final String PACKAGE = "data/";
```

Comment:
What is the sense to extract such constants and not to reuse them in the other 2 classes?